resolwe_bio/tests/processes/test_icount.py

```diff
@@ -0,0 +1,188 @@
+# pylint: disable=missing-docstring
+from resolwe.flow.models import Data
+from resolwe_bio.utils.test import BioProcessTestCase
+
+
+class ICountImport(BioProcessTestCase):
```

Comment:
@mstajdohar @tjanez This tests run the processes that require custom docker image. If Travis will be downloading both the default + custom docker image, the download time will be too large. How can we disable this tests so the Travis will not attempt to download of the "icount" image? We should than re-enable the tests when custom docker images will be smaller.